src/components/carousel/Carousel.tsx

```diff
@@ -1,10 +1,32 @@
 import {Carousel} from "@mantine/carousel"
 import { rem } from "@mantine/core"
 import { MainProduct } from ".."
+import { useSelector } from "react-redux";
+import { useEffect, useState } from "react";
 
-
+interface Product {
+  id: number;
+  name: string;
+  price: string;
+  picture_urls: string[];
+  rating: any;
```

Comment:
Why do we have "any" on rating ? We can have a number

Reply:
yeah sure!